x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/phases/cold_phase.tsx

```diff
@@ -27,15 +25,34 @@ import {
   PhaseErrorMessage,
   OptionalLabel,
   ErrableFormRow,
-  MinAgeInput,
-  NodeAllocation,
   SetPriorityInput,
+  MinAgeInput,
+  DataTierAllocation,
+  DescribedFormField,
+  NodesDataProvider,
+  DefaultAllocationWarning,
 } from '../components';
 
 const freezeLabel = i18n.translate('xpack.indexLifecycleMgmt.coldPhase.freezeIndexLabel', {
   defaultMessage: 'Freeze index',
 });
 
+const i18nTexts = {
+  defaultAllocationNotAvailable: {
+    title: i18n.translate(
+      'xpack.indexLifecycleMgmt.warmPhase.dataTier.defaultAllocationNotAvailableTitle',
+      { defaultMessage: 'No cold tier nodes found' }
+    ),
+    body: i18n.translate(
+      'xpack.indexLifecycleMgmt.warmPhase.dataTier.defaultAllocationNotAvailableBody',
+      {
+        defaultMessage:
+          'This policy will not complete allocation because no node was found for data allocation.',
```

Comment:
Switching to past-tense makes this hard to follow. Maybe:

```suggestion
          'This policy will not complete allocation because there are no cold nodes.',
```

Or, perhaps this could be rephrased in terms of the action the user needs to take, i.e. "Assign at least one node to the cold tier."

Reply:
How about: "This policy will not complete allocation because there are no cold nodes. Assign at least one node to the cold tier."?